cfme/fixtures/pytest_selenium.py

```diff
@@ -524,7 +524,7 @@ def text(loc, **kwargs):
 
     Returns: A string containing the text of the element.
     """
-    return element(loc, **kwargs).text
+    return move_to_element(loc, **kwargs).text
```

Comment:
@seandst @mfalesni  just a double check here. Do you foresee any issues with this?